client/packages/coldchain/src/Equipment/DetailView/DetailView.tsx

```diff
@@ -64,14 +77,31 @@ export const EquipmentDetailView = () => {
 
   useEffect(() => {
     if (!data) return;
-    setDraft({ ...data });
+    setDraft({
+      ...data,
+      locationIds: draft?.locationIds
+        ? draft.locationIds
+        : data.locations.nodes.map(location => location.id),
+    });
```

Comment:
the linter would like to add `draft.locationIds` now that this is a dependency - but because the effect is setting the draft, I think this will break stuff (re-rendering!)
I would add `  // eslint-disable-next-line react-hooks/exhaustive-deps` to show that the exclusion is deliberate